cmd/osbuild-composer-dbjobqueue-tests/main_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
 // +build integration
```

Comment:
My IDE insists on this, I did not fight it - I see few places where you still have this.

Reply:
Sure, `+build` will get removed when we switch to Go 1.18 completely (we cannot yet due to Fedora 35).